packages/shared/edge-utils/src/app.ts

```diff
@@ -14,9 +14,9 @@ import { EdgeStickyBucketService } from "./stickyBucketService";
 import { Request, Response } from "express";
 
 export async function edgeApp<Req, Res>(
-  context: Context<Req, Res>,
-  req: Req,
-  res: Res,
+  context: Context<unknown, unknown>,
+  req: Request,
+  res: Response,
```

Comment:
we cant use Request, Response from express since that will not work with any other service